pkg/cli/admin/catalog/mirror_test.go

```diff
@@ -101,7 +102,7 @@ func TestWriteToMapping(t *testing.T) {
 	}
 }
 
-func TestGetRegistryMapping(t *testing.T) {
+func TestGetRegistryMappingICSP(t *testing.T) {
```

Comment:
nit: Would be better to add `TODO` about safely removing this test when flags will be removed.